packages/web-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -69,20 +69,26 @@ const Dropdown = (props: SpiritDropdownProps) => {
     children,
   );
 
-  useDeprecatedMessage({
+  useDeprecationMessage({
+    method: 'property',
     trigger: !!breakpoint,
     componentName: 'Dropdown',
-    deprecatedPropName: 'breakpoint',
-    newPropName: 'fullWidthMode',
+    propertyProps: {
```

Comment:
Sounds weird...

Reply:
We plan to improve this hook in the future.